raycicmd/converter.go

```diff
@@ -71,25 +71,24 @@ func (c *converter) convertStep(step map[string]any) (
 	return c.defaultConverter.convert(step)
 }
 
-func (c *converter) convertGroup(g *pipelineGroup, filter *tagFilter) (
+func (c *converter) convertGroup(n *jobNode) (
 	*bkPipelineGroup, error,
 ) {
+	g := n.srcGroup
+
 	bkGroup := &bkPipelineGroup{
 		Group:     g.Group,
 		Key:       g.Key,
 		DependsOn: g.DependsOn,
 	}
 
-	for _, step := range g.Steps {
-		// filter steps by tags
-		if stepTags, ok := step["tags"]; ok {
-			if !filter.hit(toStringList(stepTags)) {
-				continue
-			}
+	for _, step := range n.steps {
+		if !step.include {
```

Comment:
nit: include --> isIncluded? includeSomething? to make it a boolean

Reply:
changed to "marked".